src/main.rs

```diff
@@ -786,6 +860,24 @@ async fn main() {
     let connection_pool = r2d2::Pool::builder()
         .build(ConnectionManager::<PgConnection>::new(connections_string))
         .expect("Failed to create pool.");
+    let expected_chain_ids: Vec<u64> = vec![1, 123, 32657, 84532, 11155111, 17000, 421614];
```

Comment:
Can we make it constant? Or maybe an enum?

Reply:
Done, resolving.